code/headers/controller_interface_c.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+#include <joystick_value_s.hpp>
+
+namespace r2d2::manual_control {
+    
+    enum buttons {button_a, button_b, button_x, button_y, dpad_up, dpad_down, dpad_left, dpad_right, bumper_left, bumper_right, menu_left, menu_right};
```

Comment:
This line is way to long. You should format this file. As the indents in this file are strange as well.